CRM/Contact/BAO/Query.php

```diff
@@ -1384,8 +1384,8 @@ public function query($count = FALSE, $sortByChar = FALSE, $groupContacts = FALS
       }
     }
     elseif ($sortByChar) {
+      CRM_Core_Error::deprecatedFunctionWarning('sort by char is deprecated - use alphabetQuery method');
       $select = 'SELECT DISTINCT LEFT(contact_a.sort_name, 1) as sort_name';
-      $from = $this->_simpleFromClause;
```

Comment:
@mattwire shouldn't we leave this $from in too ? For anyone who is calling the deprecated path

Reply:
ok